### DISCUSSION TITLE
Can't add property

### DISCUSSION THREAD

**Discussion by annhve** [JanusGraph/janusgraph#2367] [Q&A] [answered]
Hello everyone! I am trying to copy some data from Mci to Janusgraph(storage Cassandra). But I have some problem and get this Error :
**ERROR 1 --- [io-8080-exec-33] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.janusgraph.core.SchemaViolationException: Adding this property for key [ALIAS] and value [22203-tour22] violates a uniqueness constraint [aliasIndex]] with root cause**
Is this problem in the indexes?

**Comment by li-boxuan** [comment, marked as answer]
As the error message suggests, looks like you have created a composite index [aliasIndex] with the uniqueness constraint. You got the exception because the value [22203-tour22] already exists for key [ALIAS].

**Reply by annhve** [reply]
Probably It's true. Many thanks
